prototype-kafka/kafka-prototype-producer/src/main/java/com/klungerbo/streams/kafka/Main.java

```diff
@@ -0,0 +1,31 @@
+package com.klungerbo.streams.kafka;
+
+import com.klungerbo.streams.utils.datareceiver.DataReceiver;
+
+/**
+ * The class containing the main entry point of the Kafka producer prototype application.
+ *
+ * @version 1.0
+ * @since 1.0
+ */
+public class Main {
+    /**
+     * The entrypoint of the application.
+     *
+     * @param args commandline arguments.
+     */
+    public static void main(String[] args) {
+        try {
+            var kafkaPrototypeProducer = new KafkaPrototypeProducer();
```

Comment:
Why is var used instead of specific type?

Reply:
There is no reason to specify the type as it is comprehensible from the right hand side of the equal sign that this object is an instance of type KafkaPrototypeProducer

Reply:
I agree, but I think stylistically we should limit the use of var.
This might be personal preference, but i prefer explicity for the types.